src/entities/AppUser.ts

```diff
@@ -36,7 +36,7 @@ export class AppUser {
   @Column({ nullable: true })
   graduationYear: string;
 
-  @ManyToOne(() => InductionClass, { nullable: true, eager: true })
+  @ManyToOne(() => InductionClass, { nullable: true, deferrable: 'INITIALLY DEFERRED' })
```

Comment:
What does the deferrable do here?

Reply:
https://hashrocket.com/blog/posts/deferring-database-constraints#:~:text=In%20PostgreSQL%2C%20constraints%20are%20assumed,behavior%20to%20change%20within%20transactions.

makes seeding easier to deal with since we're inserting events in the same transaction as users

Reply:
otherwise we'd have to write like 5 seeds inductionclass -> users -> events -> attendance/rsvp